templates/google-analytics.html

```diff
@@ -1,28 +1,28 @@
+GAcontainer: GTM-PGRDRB2
+// GA containers: MyGL: GTM-PGRDRB2 GL: GTM-MBDXM9Q
+GAproperty: UA-89730678-7 
+// GA properties GL: UA-89730678-1, MyGL: UA-89730678-7, MyGL GA4: G-88QXE1Y448
+// copy pasta website or my-loop templates/google-analytics.html
+
+
+<!-- Google Tag Manager -->
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+	new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+	j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+	'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+	})(window,document,'script','dataLayer','$GAcontainer');</script>
+<!-- End Google Tag Manager -->
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=$GAcontainer" 
+	height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- End Google Tag Manager (noscript) -->
+
 <!-- Global site tag (gtag.js) - Google Analytics Universal UA - March 2022 -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-89730678-7"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=$GAproperty"></script>
```

Comment:
I don't Jerbil is rendering this properly - my browser is sending a request to `https://www.googletagmanager.com/gtag/js?id=$GAproperty` instead of `https://www.googletagmanager.com/gtag/js?id=UA-89730678-7 `.